src/state/ledger/account.rs

```diff
@@ -72,6 +73,23 @@ impl Ord for Account {
     }
 }
 
+const MINA_SCALE: u32 = 9;
+
+fn nanomina_to_mina(num: u64) -> String {
```

Comment:
don't we want to display Nanomina? per https://github.com/Granola-Team/mina-indexer/issues/172 ? Maybe we can add a flag to switch between the two